stacker/benches/bench.rs

```diff
@@ -129,7 +131,7 @@ fn main() {
     bench_vint();
 }
 
-const HASHMAP_SIZE: usize = 1 << 15;
```

Comment:
what is this about?

Reply:
This is the initial hashmap capacity, to have a fairer comparison with `FxHashMap`.
renamed to `HASHMAP_CAPACITY`